### PR TITLE
feat: basic support Apache APISIX 2.11

### PR DESCRIPTION
**Pull request by bzp2010**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Adapt Apache APISIX version 2.11, synchronize schema to support new plugins.

**Related issues**

follow: [https://github.com/apache/apisix/pull/5567](https://github.com/apache/apisix/pull/5567)

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: api/conf/schema.json [hunk 1]
```diff
@@ -5501,10 +6814,14 @@
...
+					"protocol_name",
+					"upstream"
+				],
 				"type": "object"
 			},
 			"version": 0.1
 		}
 	}
-}
+}
```
Comment (zaunist) [inline api/conf/schema.json:6827]: Do we need a new line at end of file?

Comment (bzp2010) [reply api/conf/schema.json:6827]: Yes, I will add EOL.

File: web/src/components/Plugin/data.tsx [hunk 2]
```diff
@@ -224,4 +224,22 @@ export const PLUGIN_LIST = {
     type: PluginType.other,
     hidden: true,
   },
+  'authz-casbin': {
+    type: PluginType.authentication,
+  },
+  'ldap-auth': {
+    type: PluginType.authentication,
+  },
+  datadog: {
```
Comment (zaunist) [inline web/src/components/Plugin/data.tsx:233]:
```suggestion
  'datadog': {
```
need `'` ?

Comment (bzp2010) [reply web/src/components/Plugin/data.tsx:233]: I've tried that, but it gets automatically removed by lint when doing a commit.

Comment (juzhiyuan) [reply web/src/components/Plugin/data.tsx:233]: We don't need the `'` here 😄 It's a key for JS's Object

**Review by juzhiyuan — DISMISSED** [review]

**Review by okaybase — COMMENTED** [review]
limit-conn appears twice in schema.json, and remove here(#L6728) will be better:
api/conf/schema.json#L6728 is repeated with api/conf/schema.json#L4630

File: web/src/pages/Route/components/Step1/ProxyRewrite.tsx [hunk 3]
```diff
@@ -279,6 +279,44 @@ const ProxyRewrite: React.FC<RouteModule.Step1PassProps> = ({ form, disabled })
...
+          name={['proxyRewrite', 'method']}
+          wrapperCol={{ span: 3 }}
+          initialValue=""
+        >
+          <Select>
+            <Select.Option value="">
+              {formatMessage({ id: 'page.route.select.option.methodRewriteNone' })}
+            </Select.Option>
+            {methods.map((method) => (
+              <Select.Option value={method}>{method}</Select.Option>
```
Comment (juzhiyuan) [inline web/src/pages/Route/components/Step1/ProxyRewrite.tsx:312]: key

**Comment by bzp2010** [comment]
> limit-conn appears twice in schema.json, and remove here(#L6728) will be better: api/conf/schema.json#L6728 is repeated with api/conf/schema.json#L4630

Hi, @okaybase. The `limit-conn` problem you point out, one for the HTTP plugin and one for the Stream plugin, should indeed have two.

File: web/src/pages/Route/components/Step1/ProxyRewrite.tsx (hunk 3 above)
Comment (bzp2010) [reply web/src/pages/Route/components/Step1/ProxyRewrite.tsx]: added

File: web/src/components/Plugin/locales/zh-CN.ts [hunk 4]
```diff
@@ -112,6 +115,8 @@ export default {
   'component.pluginForm.limit-count.count.tooltip': '指定时间窗口内的请求数量阈值。',
   'component.pluginForm.limit-count.time_window.tooltip':
     '时间窗口的大小（以秒为单位），超过这个时间就会重置。',
+  'component.pluginForm.limit-count.key_type.tooltip':
+    '关键字类型，支持：var（单变量）和 var_combination（组合变量）',
   'component.pluginForm.limit-count.key.tooltip':
     '用来做请求计数的有效值。例如，可以使用主机名（或服务器区域）作为关键字，以便限制每个主机名规定时间内的请求次数。我们也可以使用客户端地址作为关键字，这样我们就可以避免单个客户端规定时间内多次的连接我们的服务。当前接受的 key 有："remote_addr"（客户端 IP 地址）, "server_addr"（服务端 IP 地址）, 请求头中的"X-Forwarded-For" 或 "X-Real-IP", "consumer_name"（consumer 的 username）, "service_id" 。',
```
Comment (okaybase) [inline web/src/components/Plugin/locales/zh-CN.ts:121]: Now the key is `Input` not `Select`, so we can remove `当前接受的 key 有："remote_addr"（客户端 IP 地址）, "server_addr"（服务端 IP 地址）, 请求头中的"X-Forwarded-For" 或 "X-Real-IP", "consumer_name"（consumer 的 username）, "service_id" 。` for `component.pluginForm.limit-count.key.tooltip`

File: web/src/components/Plugin/locales/en-US.ts [hunk 5]
```diff
@@ -120,6 +123,8 @@ export default {
   'component.pluginForm.limit-count.count.tooltip': 'The specified number of requests threshold.',
   'component.pluginForm.limit-count.time_window.tooltip':
     'The time window in seconds before the request count is reset.',
+  'component.pluginForm.limit-count.key_type.tooltip':
+    'The key type, support: "var" (single var) and "var_combination" (combine var)',
   'component.pluginForm.limit-count.key.tooltip':
     'The user specified key to limit the count, now accept those as key: "remote_addr"(client\'s IP), "server_addr"(server\'s IP), "X-Forwarded-For/X-Real-IP" in request header, "consumer_name"(consumer\'s username) and "service_id".',
```
Comment (okaybase) [inline web/src/components/Plugin/locales/en-US.ts:129]: Now the key is `Input` not `Select`, so we can just keep `The user specified key to limit the count.` for `component.pluginForm.limit-count.key.tooltip`

**Comment by okaybase** [comment]
> > limit-conn appears twice in schema.json, and remove here(#L6728) will be better: api/conf/schema.json#L6728 is repeated with api/conf/schema.json#L4630
> 
> Hi, @okaybase. The `limit-conn` problem you point out, one for the HTTP plugin and one for the Stream plugin, should indeed have two.

okay, thanks~

File: web/src/components/Plugin/locales/zh-CN.ts (hunk 4 above)
Comment (bzp2010) [reply web/src/components/Plugin/locales/zh-CN.ts]: removed

File: web/src/components/Plugin/locales/en-US.ts (hunk 5 above)
Comment (bzp2010) [reply web/src/components/Plugin/locales/en-US.ts]: removed

File: web/src/components/Plugin/locales/zh-CN.ts (hunk 4 above)
Comment (okaybase) [reply web/src/components/Plugin/locales/zh-CN.ts]: thanks~

File: web/src/components/Plugin/locales/en-US.ts (hunk 5 above)
Comment (okaybase) [reply web/src/components/Plugin/locales/en-US.ts]: thanks~

**Review by juzhiyuan — APPROVED** [review]

**Comment by bzp2010** [comment]
## Update
The PR is now almost complete and ready for review.

**Review by okaybase — APPROVED** [review]

**Review by zaunist — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**